starter/cloudrun/src/logger.ts

```diff
@@ -24,10 +24,20 @@ const defaultPinoConf = (defaultLevel: string) => ({
       }
     },
   },
+  serializers: {
+    error: logger.stdSerializers.err,
+  },
 })
 
 export default logger({
   ...defaultPinoConf(config.logger.defaultLevel),
+  enableFields: [
+    'req.context.user',
+    'req.headers.x-user-region',
+    'req.headers.x-app-info',
+    'req.headers.x-api-key',
```

Comment:
These are probably something extra, unrelated to logging.

Reply:
removed